app/controllers/stories/feeds_controller.rb

```diff
@@ -37,7 +37,8 @@ def signed_in_base_feed
       if Settings::UserExperience.feed_strategy == "basic"
         Articles::Feeds::Basic.new(user: current_user, page: @page, tag: params[:tag]).feed
       else
-        optimized_signed_in_feed
```

Comment:
I inlined this helper. We only called it in once place, it's not too long and none of the similar methods used external helpers.